src/main/java/com/urswolfer/gerrit/client/rest/GerritApiImpl.java

```diff
@@ -43,6 +46,16 @@
 public class GerritApiImpl extends GerritApi.NotImplemented implements GerritRestApi {
     private final GerritRestClient gerritRestClient;
 
+    /**
+     * Construct the REST API for managing Group information.
```

Comment:
I think you can omit this comment since this is quite boilerplate code (it's the same for all APIs below).